pkg/cmd/cli/backup/create_test.go

```diff
@@ -0,0 +1,89 @@
+/*
+Copyright 2017 the Velero contributors.
```

Comment:
2019

Reply:
Sorry, the generator just takes the header from the main file, so I forgot to update it!

Reply:
Yeah, I would like to automate this away someday.